flowcraft/generator/engine.py

```diff
@@ -150,6 +150,12 @@ class of the component.
         See :func:`NextflowGenerator._get_params_string`
         """
 
+        self.manifest = ""
+        """
+        str: Stores de manifest directives string for the nextflow pipeline.
+        See :func:`NextflowGenerator.__get_params_string`
```

Comment:
I think you meant: `_get_manifest_string`

Reply:
Sorry, it was indeed a mistake! Fixed!